embedded-graphics/src/primitives/line.rs

```diff
@@ -250,6 +279,17 @@ mod tests {
         test_expected_line(start, end, &expected);
     }
 
+    #[test]
+    fn no_stroke_width_no_line() {
+        let start = Point::new(2, 3);
+        let end = Point::new(3, 2);
+
+        let line =
+            Line::new(start, end).into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 0));
+
+        assert_eq!(line.into_iter().next(), None);
```

Comment:
Not really necessary to change this, but this made me think about what the nicest looking and most semantic way to test if an iterator is empty is:
```suggestion
        assert!(line.into_iter().eq(core::iter::empty()));
```
```suggestion
        assert_eq!(line.into_iter().count(), 0);
```